locations/spiders/kfc_za.py

```diff
@@ -0,0 +1,56 @@
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+from locations.pipelines.address_clean_up import clean_address
+from locations.spiders.kfc_us import KFC_SHARED_ATTRIBUTES
+
+
+class KfcZASpider(scrapy.Spider):
+    name = "kfc_za"
+    item_attributes = KFC_SHARED_ATTRIBUTES
+    start_urls = ["https://orderserv-kfc-eu-olo-api.yum.com/dev/v1/stores/"]
+    tenant_id = "5f89a603781e4df48d31337a6c1252a8"
+    requires_proxy = True  # May require ZA proxy, possibly residential IPs only.
```

Comment:
The error you're getting from CI is due to this `requires_proxy=True` telling the CI process to try and use a Zyte API account that has no balance remaining in it. If you're in ZA and don't need a proxy, can you provide the spider output when you omit `requires_proxy=True`?

Reply:
It works fine from ZA without `requires_proxy=True`

Reply:
It works fine from ZA without `requires_proxy=True`